specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-15-preview/images.json

```diff
@@ -0,0 +1,527 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateImagesCreation"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Created",
+            "headers": {
+              "operation-location": {
+                "type": "string",
+                "format": "url",
+                "description": "URL to poll for the operation result."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "Service generated correlation id identifying the request, in the form of a GUID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/GenerateImagesResponse"
+            }
+          },
+          "default": {
+            "description": "An error occurred.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code indicating what went wrong."
+              },
+              "x-ms-request-id": {
+                "type": "string",
+                "format": "uuid",
+                "description": "Service generated correlation id identifying the request, in the form of a GUID."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "Creating an image generation operation.": {
+            "$ref": "./examples/post_images_generate.json"
+          }
+        }
+      }
+    },
+    "/operations/{operationId}": {
+      "get": {
+        "summary": "Returns the status of the operation",
+        "operationId": "Operations_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionQueryParameter"
+          },
+          {
+            "in": "path",
+            "name": "operationId",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "The id of the operation",
+            "maxLength": 36,
+            "pattern": "^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "retry-after": {
+                "type": "integer",
+                "format": "int64",
+                "description": "Indicates the number of seconds that the client should wait before trying to get the result from the URL identified by the Operation-Location header if the operation did not terminate yet."
```

Comment:
>L identified by the Operation-Location header if the op [](http://example.com/codeflow?start=135&length=55)

rephrase, there is no operation location here.

Reply:
Fixed.